src/types/smart_contracts.rs

```diff
@@ -2,6 +2,7 @@
 
 use super::hashes;
 use crate::constants::*;
+pub use concordium_contracts_common;
```

Comment:
Please provide a doc comment for this re-export describing the reasons and functionality therein.

Reply:
I have added a doc comment